lockapi.pm

```diff
@@ -72,10 +72,13 @@ sub mutex_try_lock {
 }
 
 sub mutex_unlock {
-    my ($name) = @_;
+    my ($name, $where) = @_;
+    my $param = {action => 'unlock'};
+    $param->{where} = $where if $where;
+
     bmwqemu::mydie('missing lock name') unless $name;
     bmwqemu::diag("mutex unlock '$name'");
-    my $res = api_call('post', "mutex/$name", {action => 'unlock'})->code;
```

Comment:
where is all this documented btw?

Reply:
I think all documentation we have is [here](https://github.com/os-autoinst/openQA/blob/master/docs/WritingTests.asciidoc#test-synchronization-and-locking-api).

Reply:
small improvement in https://github.com/os-autoinst/openQA/pull/1686